app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -103,6 +102,35 @@ function onPlotRelatedGenes() {
   window.SCP.log('ideogram:related-genes', props)
 }
 
+/**
+ * Set inner visibility of ideogram
+ *
+ * @param found Boolean Whether related genes have been found
+ */
+function setFoundRelatedGenes(found) {
+  const innerFoundClass = (found) ? 'found-related-genes' : ''
+  const innerSelect = '#related-genes-ideogram-container #_ideogramInnerWrap'
+  document.querySelector(innerSelect).classList = innerFoundClass
+
+  if (found) {
+    const ideoContainer =
+      document.querySelector('#related-genes-ideogram-container')
```

Comment:
I don't understand why you need to change classes on two different elements to make the ideogram visible.  Shouldn't showing/hiding the container be enough?

Reply:
Good catch, done in 9e52773.